app/util/AWS.scala

```diff
@@ -34,6 +34,8 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val kinesisStreamName = config.getString("aws.kinesis.streamName").get
 
+  lazy val kinesisReindexStreamName = config.getString("aws.kinesis.reindexStreamName").get
```

Comment:
@paulmr FYI we decided that for all atoms we would use a single kinesis stream for indexing and reindexing (which are already defined, see here: https://github.com/guardian/content-api/blob/master/porter/src/main/scala/com.gu.contentapi.porter/Config.scala#L68)

So you may just use this one. Was this what you was planning?